assets/js/modules/analytics/components/setup/SetupFormGA4.js

```diff
@@ -16,23 +16,57 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { useMount } from 'react-use';
+
 /**
  * WordPress dependencies
  */
+import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { AccountSelect } from '../common';
+import Data from 'googlesitekit-data';
+import { CORE_FORMS } from '../../../../googlesitekit/datastore/forms/constants';
+import { STORE_NAME, PROPERTY_CREATE, FORM_SETUP } from '../../datastore/constants';
+import StoreErrorNotices from '../../../../components/StoreErrorNotices';
+import PropertySelect from '../../../analytics-4/components/common/PropertySelect';
+import AccountSelect from '../common/AccountSelect';
+import GA4PropertyNotice from '../common/GA4PropertyNotice';
+const { useSelect, useDispatch } = Data;
 
 export default function SetupFormGA4() {
+	const accounts = useSelect( ( select ) => select( STORE_NAME ).getAccounts() ) || [];
+
+	const { selectProperty } = useDispatch( STORE_NAME );
+	const { setValues } = useDispatch( CORE_FORMS );
+	useMount( () => {
+		selectProperty( PROPERTY_CREATE );
+		setValues( FORM_SETUP, { profileName: 'All Web Site Data' } );
+	} );
+
 	return (
 		<Fragment>
+			<StoreErrorNotices moduleSlug="analytics" storeName={ STORE_NAME } />
+
+			{ ( !! accounts.length ) && (
+				<p className="googlesitekit-margin-bottom-0">
+					{ __( 'Please select the account information below. You can change this view later in your settings.', 'google-site-kit' ) }
+				</p>
+			) }
+
 			<div className="googlesitekit-setup-module__inputs">
 				<AccountSelect />
+				<PropertySelect />
 			</div>
-			<div>SetupFormGA4</div>
+
+			<GA4PropertyNotice
+				notice={ __( 'An associated Universal Analytics property will also be created.', 'google-site-kit' ) }
+			/>
```

Comment:
I realize this isn't from this PR but I noticed there are some inconsistencies with this notice and the design, specifically around the "learn more" link:

**Current**
![image](https://user-images.githubusercontent.com/1621608/119956293-9ed3f000-bfa9-11eb-9d94-7e8884307948.png)

**Design**
![image](https://user-images.githubusercontent.com/1621608/119956334-aabfb200-bfa9-11eb-9e24-645c0c6daff8.png)

cc: @asvinb